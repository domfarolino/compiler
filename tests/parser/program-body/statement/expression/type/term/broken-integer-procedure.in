program dominic is

  integer result;

  procedure f()
  begin
  end procedure;

begin

  result := 10 * f;

end program.
