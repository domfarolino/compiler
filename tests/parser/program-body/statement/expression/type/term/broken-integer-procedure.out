[1;31mLine 11: [Term]: integer and procedure cannot be multiplied or divided
Line 11: Error parsing program body
[0m